src/AppComponent.hpp

```diff
@@ -15,34 +15,49 @@
  * 
  * This project is a derivative work from oatpp-web-starter
  */
-
 #ifndef AppComponent_hpp
 #define AppComponent_hpp
 
 #include "oatpp/web/server/HttpConnectionHandler.hpp"
-#include "oatpp/network/server/SimpleTCPConnectionProvider.hpp"
-#include "oatpp/parser/json/mapping/ObjectMapper.hpp"
-#include "oatpp/core/macro/component.hpp"
 
-const int PORT = 3000;
+#include "oatpp/network/tcp/server/ConnectionProvider.hpp"
 
+#include "oatpp/parser/json/mapping/ObjectMapper.hpp"
 
+#include "oatpp/core/macro/component.hpp"
+
+/**
+ *  Class which creates and holds Application components and registers components in oatpp::base::Environment
+ *  Order of components initialization is from top to bottom
+ */
 class AppComponent {
 public:
-
+  
+  /**
+   *  Create ConnectionProvider component which listens on the port
+   */
   OATPP_CREATE_COMPONENT(std::shared_ptr<oatpp::network::ServerConnectionProvider>, serverConnectionProvider)([] {
-    return oatpp::network::server::SimpleTCPConnectionProvider::createShared(PORT);
+    return oatpp::network::tcp::server::ConnectionProvider::createShared({"0.0.0.0", 8000, oatpp::network::Address::IP_4});
```

Comment:
Can you keep the port in a constant, it should be provided via Env Var (also let's use 3000 which si our standard port)

Reply:
Providing it by Env Var seems too tricky for the actual usage. I put it in a costant